### PR TITLE
Redefine apim analytics scopes

### PR DESCRIPTION
**Pull request by ruwiniwj**
Fixes https://github.com/wso2/analytics-apim/issues/1127

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/resources/tenant/tenant-conf.json [hunk 1]
```diff
@@ -158,28 +158,52 @@
...
       },
       {
-        "Name": "apim_analytics:everyone",
-        "Roles": "Internal/everyone"
+        "Name": "apim_analytics:manager_dashboard:view",
+        "Roles": "admin,Internal/analytics"
+      },
+      {
+        "Name": "apim_analytics:publisher_dashboard:own",
+        "Roles": "Internal/creator,Internal/publisher"
```
Comment (RAVEENSR) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/resources/tenant/tenant-conf.json:186]: Add `admin` role to all the scopes

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/resources/tenant/tenant-conf.json [hunk 2]
```diff
@@ -158,28 +158,52 @@
...
+      },
+      {
+        "Name": "apim_analytics:publisher_dashboard:edit",
+        "Roles": "Internal/creator,Internal/publisher"
```
Comment (TanyaM) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/resources/tenant/tenant-conf.json:190]: Shouldn't we allow these two scopes only to admin users? For example if one publisher accidentally deletes the dashboard, all the other publishers won't be able to see the dashboard

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/resources/tenant/tenant-conf.json [hunk 3]
```diff
@@ -158,28 +158,52 @@
...
+        "Name": "apim_analytics:publisher_dashboard:view",
+        "Roles": "Internal/creator,Internal/publisher"
+      },
+      {
+        "Name": "apim_analytics:subscriber_dashboard:own",
+        "Roles": "Internal/subscriber"
+      },
+      {
+        "Name": "apim_analytics:subscriber_dashboard:edit",
+        "Roles": "Internal/subscriber"
```
Comment (TanyaM) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/resources/tenant/tenant-conf.json:202]: same goes here.

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/resources/tenant/tenant-conf.json (hunk 1 above)
Comment (ruwiniwj) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/resources/tenant/tenant-conf.json]: fixed with 5a3ee18224c414b9f77a23a17c9b41da6ddc1f37

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/resources/tenant/tenant-conf.json (hunk 2 above)
Comment (ruwiniwj) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/resources/tenant/tenant-conf.json]: fixed with 5a3ee18224c414b9f77a23a17c9b41da6ddc1f37

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/resources/tenant/tenant-conf.json (hunk 3 above)
Comment (ruwiniwj) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/resources/tenant/tenant-conf.json]: fixed with 5a3ee18224c414b9f77a23a17c9b41da6ddc1f37

**Pull request merged**